.github/workflows/release-nix.yml

```
@@ -0,0 +1,58 @@
+name: "Release nix"
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  nix-flake-release:
+    name: 'Nix flake release'
+    strategy:
+      matrix:
+        include:
+          - runner: ubuntu-latest
+            os: ubuntu-latest
+          - runner: macos-12
+            os: macos-12
+          - runner: MacM1
+            os: self-macos-12
+    runs-on: ${{ matrix.runner }}
+    steps:
+      - name: 'Check out code'
+        uses: actions/checkout@v3.0.2
+
+      - name: 'Upgrade bash'
+        if: ${{ contains(matrix.os, 'macos') }}
+        run: brew install bash
+
+      - name: 'Install Nix'
+        if: ${{ !startsWith(matrix.os, 'self') }}
+        uses: cachix/install-nix-action@v15
+        with:
+          extra_nix_config: |
+            substituters = http://cache.nixos.org https://cache.iog.io
+            trusted-public-keys = cache.nixos.org-1:6NCHdD59X431o0gWypbMrAURkbJ16ZPMQFGspcDShjY= hydra.iohk.io:f/Ea+s+dFdN+3Y/G+FDgSq+a5NEWhJGzdjvKNGv0/EQ=
+
+      - name: 'Install Cachix'
+        if: ${{ !startsWith(matrix.os, 'self') }}
+        uses: cachix/cachix-action@v10
+        with:
+          name: k-framework
+          signingKey: '${{ secrets.RV_DEVOPS_CACHIX_TOKEN }}'
+
+      - name: 'Build and cache K Framework'
+        run: |
+          export GC_DONT_GC=1
+          export CACHIX_AUTH_TOKEN=$RV_DEVOPS_CACHIX_TOKEN
+          nix shell nixpkgs#jq
+          nix-store --query --requisites --include-outputs \
+            $(nix-store --query --deriver \
+              $(nix build .#k --json | jq -r '.[].outputs | to_entries[].value')) \
+            | cachix push k-framework
```

I'd rather format this like this (for readabliity, and to make sure that it's more clear which subcommand fails if one of them fails):

```suggestion
          deriver=$(nix build .k --json | jq -r '.[].outputs | to_entries[].value')
          outputs=$(nix-store --query --deriver ${deriver})
          nix-store --query --requisites --include-outputs ${outputs} | cachix push k-framework
```

Same with the other steps in this job which use this pattern.